org.freedesktop.LinuxAudio.BaseExtension.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id" : "org.freedesktop.LinuxAudio.BaseExtension",
+  "branch": "20.04",
```

Comment:
I'd rather have it match runtime version it targets for my own sanity.

Reply:
done (at the next update)